.github/workflows/ci.yml

```diff
@@ -115,3 +115,33 @@ jobs:
             gem install bundler
             bundle install
             bundle exec rake ${{ matrix.task }}
+
+  openbsd:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - uses: vmactions/openbsd-vm@v1
```

Comment:
Although I knew about `vmactions/freebsd-vm` I did not know there were other BSD variations available!